microsoft/vscode-python-devicesimulator
# Sensors for clue

# Description:
Make all the sensors for clue. 
[AB#34292](https://microsoftgarage.visualstudio.com/002806e2-ebaa-4672-9d2e-5fe5d29154ef/_workitems/edit/34292)
[AB#34296](https://microsoftgarage.visualstudio.com/002806e2-ebaa-4672-9d2e-5fe5d29154ef/_workitems/edit/34296)
[AB#34297](https://microsoftgarage.visualstudio.com/002806e2-ebaa-4672-9d2e-5fe5d29154ef/_workitems/edit/34297)
 List of sensors: Temperature, Light accelerometer, Leds, push_button, gpio, speaker, sound , pressure, humidity, gesture, proximity
## Type of change

**Testing Steps**
1. Run the code to display sensor values 
```
from adafruit_bitmap_font import bitmap_font
from adafruit_clue import clue

clue_data = clue.simple_text_display(text_scale=2)

while True:
    clue_data[0].text = "Acceleration: {:.2f} {:.2f} {:.2f} m/s^2".format(
        *clue.acceleration
    )
    clue_data[1].text = "Pressure: {:.3f} hPa".format(clue.pressure)
    clue_data[2].text = "Altitude: {:.1f} m".format(clue.altitude)
    clue_data[3].text = "Temperature: {:.1f} C".format(clue.temperature)
    clue_data[4].text = "Humidity: {:.1f} %".format(clue.humidity)
    clue_data[5].text = "Proximity: {}".format(clue.proximity)
    clue_data[6].text = "Gesture: {}".format(clue.gesture)
    clue_data[7].text = "Color: R: {} G: {} B: {} C: {}".format(*clue.color)
    clue_data[8].text = "Button A: {}".format(clue.button_a)
    clue_data[9].text = "Button B: {}".format(clue.button_b)

    clue_data.show()
```

2. Modify sensors with Value to display the values on the clue screen. Gesture should return an integer depending on it's mapped value (ex: UP=1)
